src/iOS/Avalonia.iOS/Avalonia.iOS.csproj

```diff
@@ -1,17 +1,11 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>xamarin.ios10</TargetFramework>
+    <TargetFramework>net6.0-ios</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <LangVersion>latest</LangVersion>
+    <SupportedOSPlatformVersion>10.0</SupportedOSPlatformVersion>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="OpenTK-1.0" />
```

Comment:
OpenTK GL isn't available in net6-ios, and Xamarin team seems to migrate away from having out-of-box OpenTK, deprecating it slowly.
There is another option of having [OpenTK.Graphics](https://github.com/opentk/opentk) nuget package.
But we already have these mappings in GlInterface class used for other backends, so why not use it here.